frontend/src/const/routes.ts

```diff
@@ -298,6 +298,10 @@ export const ROUTE_PATHS = Object.freeze({
     SshCaByIDPage: setRoute(
       "/ssh/$projectId/ca/$caId",
       "/_authenticate/_inject-org-details/_org-layout/ssh/$projectId/_ssh-layout/ca/$caId"
+    ),
+    SshHostGroupDetailsByIDPage: setRoute(
+      "/ssh/$projectId/ssh-host-groups/$sshHostGroupId",
+      "/_authenticate/_inject-org-details/_org-layout/ssh/$projectId/_ssh-layout/ssh-host-groups/$sshHostGroupId"
```

Comment:
(ignore placement of comment)

When I update the login mapping of a host group, it doesn't invalidate the login mapping display of the hosts

<img width="1512" alt="image" src="https://github.com/user-attachments/assets/67fce7ec-2976-430b-905e-f8b35b8c1e20" />

Reply:
Fixed.